packages/scss/src/commons/utils/a11y.scss

```diff
@@ -11,7 +11,8 @@
 	contain: paint #{$suffix};
 }
 
-@mixin focusVisible($offset: 2px) {
```

Comment:
En forçant le border-radius à 0 par défaut lors de l'appel du mixin, tu empêches de pouvoir hériter d'un éventuel border-radius déjà présent sur l’élément. 

Je te propose ça à la place : 
```
@if $radius != false {
  border-radius: $radius;
} 
```